vignettes/calibration-2021.Rmd

```diff
@@ -0,0 +1,292 @@
+---
+
+```{r, include = FALSE}
+library(tidyverse)
+library(plotly)
+
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Objective
+
+We are using genetic algorithms, a type of stochastic optimization, to calibrate the following model coefficients:
+
+* `..surv_adult_enroute_int` used in submodel `surv_adult_enroute`
+* `..surv_juv_rear_int` used in submodel `surv_juv_rear`
+* `..surv_juv_rear_contact_points` used in submodel `surv_juv_rear`
+* `..surv_juv_rear_prop_diversions` used in submodel `surv_juv_rear`
+* `..surv_juv_rear_total_diversions` used in submodel `surv_juv_rear`
+* `..surv_juv_bypass_int` used in submodel `surv_juv_bypass`
+* `..surv_juv_delta_int` used in submodel `surv_juv_delta`
+* `..surv_juv_delta_contact_points` used in submodel `surv_juv_delta`
+* `..surv_juv_delta_total_diverted` used in submodel `surv_juv_delta`
+* `..surv_juv_outmigration_sj_int` used in submodel `surv_juv_outmigration_san_joaquin`
+* `..ocean_entry_success_int`  used in submodel `ocean_entry_success`
+
+A detailed description for each of these parameters can be found in the 
+[Winter Run Documentation Website](https://cvpia-osc.github.io/winterRunDSM/reference/params.html)
+
+We are restricting the calibration to watersheds with Grandtab data, these are:
+
+```{r, echo=FALSE}
+keep <- c(1, 3)
+knitr::kable(winterRunDSM::watershed_attributes[keep, "watershed", drop = FALSE])
+```
+
+
+## Packages
+
+```{r, message=FALSE}
+library(GA)
+library(winterRunDSM)
+library(DSMCalibrationData)
+library(tidyverse)
+```
+
+We are using the [GA](http://luca-scr.github.io/GA/index.html) R package to implement our genetic algorithms search to calibrate the above intercepts for the `fallRunDSM::winter_run_model()` function of the [winterRunDSM](https://cvpia-osc.github.io/winterRunDSM) R package.
+
+Additionally, we are using [DSMCalibrationData](https://github.com/CVPIA-OSC/DSMCalibrationData) to access calibration related data.
+
+## Calibrating Data
+
+### GrandTab
+
+We will be minimizing the error between [GrandTab](https://wildlife.ca.gov/Conservation/Fishes/Chinook-Salmon/Anadromous-Assessment) estimated escapement data for the years 1998-2017 to natural spawners predicted by the DSM. We are further scaling this error by a weight representing the amount of observed data available for a given watershed. Given that
+GrandTab includes both hatchery and natural spawners, we will scale GrandTab escapement
+values using an estimated proportion natural spawners for each watershed.
+This will allow us to compare predicted natural spawners to psuedo GrandTab natural spawners.
+
+For calibration, we have prepared the GrandTab data in two ways:
+
+1. `DSMCalibrationData::grandtab_observed` - This data will be used in the fitness function to measure the difference between model predictions and observed escapement. Missing values are NA and we made all records less than 100 NA to account for a lack of confidence for counts less than 100. 
```

Comment:
when the vignette is rendered these are displayed as clickable links but they lead to no site, can you check if there is  a way we can link to the docs page of `DSMCalibrationData::grandtab_observed` ?

Reply:
No docs page for DSMCalibrationData::grandtab_observed; will push it for another time